packages/flex-plugin-scripts/README.md

```diff
@@ -10,6 +10,64 @@ Scripts to run, build, and deploy Flex plugin.
 
 This repo will automatically be installed as part of creating a new plugin.
 
+## Scripts
+
+### Release
+
+```bash
+npm run release {major, minor, patch, custom, overwrite} {--public}
+```
+
+This script will build, and then deploy your plugin bundle to Twilio Assets. Please note that your accountSid and authToken are required to use this script. Your plugins will be uploaded to Twilio Assets using the following format:
+
+```text
+https://domain.twil.io/plugins/PluginName/Version/bundle.js
+```
+
+Where `PluginName` is the `name` from `package.json`, and `Versions` is the `version` from `package.json`. For example:
+
+```text
+https://domain.twil.io/plugins/dialpad/1.0.0/bundle.js
+https://domain.twil.io/plugins/dialpad/1.1.0/bundle.js
+https://domain.twil.io/plugins/dialpad/1.1.1/bundle.js`
+```
+
+You may choose between `major, minor, patch, custom, overwrite` options:
```

Comment:
might be useful to provide examples for each of these (like you did for custom) just to be very clear

Reply:
my bad, i thought the other commands also required versions. you can revert since they're not really necessary if it's just `npm run release {type}`